app/select.js

```diff
@@ -0,0 +1,52 @@
+define(['app/database', 'jquery', 'parsley', 'bootstrap-select'], function (db, $, parsley, select) {
+  return {
+    initiate: function(name, item) {
+      switch (name) {
+        case 'languages':
+          // Load languages from database
+          db.each("getLanguages", "", function (row,err) {
+            $(item).append($("<option></option>")
+              .attr("value",row.language_id)
+              .text(row.language_name));
+          });
+          break;
+        case 'subjects':
+          // Load subjects from database
+          var rows = db.getQuery('getUserSubjects',[]);
+          // Add an option for each subject to the dropdown
+          for (var i = 0 ; i < rows.length; i++) {
+            var row = rows[i];
+            $(item).append($("<option></option>")
+              .attr("value",row.subject_id)
+              .text(row.subject_name));
+          }
+          break;
+
```

Comment:
You might want to add a `default` case that logs an error message, to avoid spending hours searching for a small typo.